src/plugins/blocksmith/components/instagram-media-block-modal/validateInstagramMediaId.test.jsx

```diff
@@ -1,34 +1,59 @@
 import test from 'tape';
```

Comment:
@jalopez13 this file has wrong extension, and should be named after the file it is testing

Reply:
@efjacobson 

Updated extension from .jsx to .js.